amr-wind/turbulence/LES/OneEqKsgs.cpp

```diff
@@ -48,8 +48,12 @@ OneEqKsgsM84<Transport>::OneEqKsgsM84(CFDSim& sim)
     {
         amrex::ParmParse pp("ABL");
         pp.get("reference_temperature", m_ref_theta);
+        pp.query("enable_hybrid_rl_mode", m_hybrid_rl);
     }
 
+    if (m_hybrid_rl)
+        m_sdr = &(sim.repo().declare_field("sdr", 1, 1, 1));
```

Comment:
Overset connectivity with TIOGA currently requires the _same number of ghost cells_ for all fields being exchanged. Use the same number of ghost cells as TKE field here. 

```C++
this->m_sdr = &(sim.repo().declare_field("sdr", 1, this->m_tke.num_grow()[0], 1);
```

Also if we decide to keep `fillpatch` we should also set default fillpatch operator 

```C++
this->m_sdr->set_default_fill_patch_op(this->m_sim.time());
```

Reply:
Fixed in 8ecbe368e5e6815e45ca2be481488572dfd0ea7f